### ISSUE TITLE
[Feature request] Add User Agent changer

### ISSUE THREAD

**Issue by h1toru** [s1t3m4p/vroxy#1]
I did tests on https://vytal.io/scan and Vytal is the only extension that gives the same result on User Agent string. Other extensions like [Random User-Agent (Switcher)](https://github.com/tarampampam/random-user-agent), [Browser Boost](https://github.com/BrowserBoost/Extension) gives different result on Top window&Frame, and Web worker, example: tarampampam/random-user-agent#624.

I know this is an alternative to Vytal, that's why I hope it has User Agent changer feature like Vytal does.

**Comment by deathtrip** [comment]
I agree, the user agent switcher was the feature i used most in vytal, even more than the location/timezone spoofer.

**Comment by s1t3m4p** [comment]
Its possible I add it in the future when I have time, but no plans right now.
